tests/virt_autotest/libvirt_isolated_virtual_network.pm

```diff
@@ -62,10 +62,13 @@ sub run_test {
 
         assert_script_run("virsh attach-interface $guest network vnet_isolated --model $model --mac $mac --live $affecter", 60);
 
-        my $net = (get_var('XEN') || check_var('SYSTEM_ROLE', 'xen') || check_var('HOST_HYPERVISOR', 'xen') && is_sle('=11-sp4')) ? 'netfront' : 'vnet_isolated';
+        my $net = is_sle('=11-sp4') ? 'br123' : 'vnet_isolated';
         test_network_interface($guest, mac => $mac, gate => $gate, isolated => 1, net => $net);
 
         assert_script_run("virsh detach-interface $guest --mac $mac $exclusive");
+
+        #cleanup file /etc/hosts after finished virtual network test
+        script_run "sed -i '/$guest/d' /etc/hosts";
```

Comment:
You want to restore /etc/hosts or remove all $guest records in it ? I think you can recover from a backed up /etc/hosts.

Reply:
yes, during virtual network testing, there was modified file /etc/hosts depended on virtual network required, to keep connection both on vm host and guest system via ssh. So, after finished virtual network testing, need to cleanup ile /etc/hosts as required. 
Great, will updated code here.

Reply:
@guoxuguang Seems this is the only one not updated. What's the plan?

Reply:
I am working on this comments now.
Will updated this assp.

Reply:
Had updated code here. 

Add the latest Verification run:
guest sles15sp2 on host sles11sp4 kvm
http://10.67.19.82/tests/605

Reply:
@alice-suse @waynechen55 please help me make sure if there was any comments for this PR.

Reply:
@guoxuguang  If no more comments from @waynechen55  about this change, will merge it this afternoon.